test/fixtures/get_balance.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+exports.rootExpectedKeys = [
+	'success', 'ledger', 'validated', 'balances'
```

Comment:
I see your intent here with these kind of fixtures. However, if the structure of response changes (i.e new property with child properties) we won't be able to know. My suggestions is to mimic the json object response and asserting with a deepEqual.